bin/beepmein

```diff
@@ -0,0 +1,281 @@
+#!/bin/bash
+
+# beepmein
+
+# A little wrapper around linux 'at' command, that will beep you at specific time, see --help.
+# On Debian systems make sure 'at' is installed with --no-install-recommends.
+
+# Copyright (C) 2019/2020 brontosaurusrex
+# License: GPLv3 or later.
+
+# prepare config dir
+config="$HOME/.cache/beepmein"
+mkdir -p "$config"
```

Comment:
Check for errors; might fail (HOME not matching user, for example with nested sudo/su invocation).

Reply:
Would `set -e` work for the entire script?

Reply:
I guess -e will not work, due to various constructs like 

    playa "$alarmSample" || alarmfailsafe

Reply:
Opinions are divided on `set -e` 
It's commonly found in Debian scripts, but some say it's better to check every command for errors individually.

Either way, it's a Good Thing to consider the consequences of a failed command, and cover it if necessary.